src/elements/content-sidebar/Sidebar.js

```diff
@@ -91,7 +93,11 @@ const Sidebar = ({
             />
         )}
         {selectedView === SIDEBAR_VIEW_METADATA && (
-            <MetadataSidebar fileId={fileId} {...metadataSidebarProps} startMarkName={MARK_NAME_JS_LOADING_METADATA} />
+            <LoadableMetadataSidebar
+                fileId={fileId}
+                {...metadataSidebarProps}
+                startMarkName={MARK_NAME_JS_LOADING_METADATA}
```

Comment:
Is the mark name needed here? Others dont have it

Reply:
Yes, all sidebars have this. It is passed and consumed to a HoC which wraps each sidebar container (withLogger)